### PR TITLE
deploy: Updates to enable/ease deployment and site refresh

### PR DESCRIPTION
**Pull request by atla5**
### Description of Changes 
- updates to deploy script
  - add script variables to enable SFTP and directories
  - remove `gh-pages` branch switching (not using github.io hosting anyway)
  - ignore the noise generated by switching between environments via `git stash` after `build`
- update README:
  - better reflect documentation (or at least don't oversell it. :p ) 
  - describe script usage
- COPY
  - add some new 2018 Events
  - fix the DMPTool link
  - DDC -> BEDAC

### CODE REVIEW

File: _site/telegraph/dw-rebuild.sh [hunk 1]
```diff
@@ -1,57 +1,48 @@
...
+local_path=~/projects/dw/ # [[ ADD REPO LOCATION HERE ex ~/Documents/github/bulib/dataWebsite]]
 local_build=_site
 
-server= [[ ADD SERVER LOCATION HERE ]]
+server=webdev.bu.edu # [[ ADD SERVER LOCATION HERE ]]
 
-dev_path= [[ ADD DEV SERVER PATH HERE ]] 
-production_path= [[ ADD PRODUCTION PATH HERE]]
+dev_path=/web/d/e/dev/data  # [[ ADD DEV SERVER PATH HERE ]]
+production_path=/web/d/a/data  # [[ ADD PRODUCTION PATH HERE]]
```
Comment (atla5) [inline _site/telegraph/dw-rebuild.sh:16]: defined paths

File: _site/telegraph/dw-rebuild.sh [hunk 2]
```diff
@@ -61,46 +52,28 @@ do
...
-
-
-
+EOF"
 
+echo "--- CLEANING UP... ---"
+if ! git diff-index --quiet HEAD --; then
+    echo "BE ADVISED: changes detected! stashing and dropping them..."
+    eval "git stash"
+    eval "git stash drop"
```
Comment (atla5) [inline _site/telegraph/dw-rebuild.sh:74]: added cleanup

**Comment by tomhohenstein** [comment]
Nice to see this being worked on! The updates look good. If you ever have questions or think "what was that guy thinking?!?!" I'm happy to help.

**Comment by atla5** [comment]
thank you very much! i definitely appreciate the resource, and nice work on the site.

**Pull request merged**